### PR TITLE
AutoCorrect fix files/zh-cn/web/{e,f,g}*

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

File: files/zh-cn/web/api/element/setattribute/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ element.setAttribute(name, value);
 
 布尔属性（原文是 Boolean attributes）只要出现在元素上就会被认为是 `true` ，无论它的值是什么; 一般来说，你应该将 `value` 设置为空字符串 (`""`) 。(一些人使用这个属性的名称作为值; 这不会出现什么问题，但这是不规范的). See the [example](#示例) below for a practical demonstration.
 
-由于将指定的值转换为字符串，因此指定 null 不一定能达到您的期望。 而不是删除属性或将其值设置为{{jsxref("null")}}，而是将属性的值设置为字符串“ null”。 如果要删除属性，请调用{{domxref("Element.removeAttribute”，“ removeAttribute（）")}}}。
+由于将指定的值转换为字符串，因此指定 null 不一定能达到您的期望。而不是删除属性或将其值设置为{{jsxref("null")}}，而是将属性的值设置为字符串“null”。如果要删除属性，请调用{{domxref("Element.removeAttribute”，“removeAttribute（）")}}}。
```
Comment (nschonni) [inline files/zh-cn/web/api/element/setattribute/index.md:33]: I don't think the macros will work with the smart quotes. Not your change, just something left over
```suggestion
由于将指定的值转换为字符串，因此指定 null 不一定能达到您的期望。而不是删除属性或将其值设置为{{jsxref("null")}}，而是将属性的值设置为字符串“null”。如果要删除属性，请调用{{domxref("Element.removeAttribute"，"removeAttribute()")}}}。
```

File: files/zh-cn/web/api/element/getelementsbytagname/index.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ slug: Web/API/Element/getElementsByTagName
 elements = element.getElementsByTagName(tagName)
 ```
 
-- `elements 搜索到的元素的动态 HTML 集合`{{domxref("HTMLCollection")}}，它们的顺序是`在子树中出现的顺序`。 如果没有搜索到元素则这个集合为空。
+- `elements 搜索到的元素的动态 HTML 集合`{{domxref("HTMLCollection")}}，它们的顺序是`在子树中出现的顺序`。如果没有搜索到元素则这个集合为空。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/getelementsbytagname/index.md:20]:
```suggestion
- `elements` 搜索到的元素的动态 HTML 集合 {{domxref("HTMLCollection")}}，它们的顺序是在子树中出现的顺序。如果没有搜索到元素则这个集合为空。
```

File: files/zh-cn/web/api/element/queryselectorall/index.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ const refs = [...document.querySelectorAll(`[data-name*="funnel-chart-percent"]`
 // const refs = [...document.querySelectorAll(`[class~="funnel-chart-percent"]`)];
 ```
 
-下面的例子返回了 HTML 文档中的`body`元素的所有`p后代`元素：
+下面的例子返回了 HTML 文档中的`body`元素的所有`p 后代`元素：
```
Comment (yin1999) [inline files/zh-cn/web/api/element/queryselectorall/index.md:47]:
```suggestion
下面的例子返回了 HTML 文档中的 `body` 元素的所有 `p` 后代元素：
```

File: files/zh-cn/web/api/element/setattributenodens/index.md [hunk 4]
```diff
@@ -36,9 +36,9 @@ alert(d2.attributes[1].value) // returns: `utterleft'
 
 ## Notes
 
-如果指定的属性在元素上存在， 接着此属性被新的属性替换的话被替换的属性会被返回。
+如果指定的属性在元素上存在，接着此属性被新的属性替换的话被替换的属性会被返回。
 
-注意：如果你尝试设置的时候没有克隆那个节点，Mozia 会抛出一个 NS_ERROR_DOM_INUSE_ATTRIBUTE_ERR ："Attribute already in use" 错误， 因为 DOM 需要克隆属性之后才能重复使用 ( 不像其他节点一样可以被删除 ) 。
+注意：如果你尝试设置的时候没有克隆那个节点，Mozia 会抛出一个 NS_ERROR_DOM_INUSE_ATTRIBUTE_ERR："Attribute already in use" 错误，因为 DOM 需要克隆属性之后才能重复使用 ( 不像其他节点一样可以被删除 ) 。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/setattributenodens/index.md:41]:
```suggestion
注意：如果你尝试设置的时候没有克隆那个节点，Mozia 会抛出一个 NS_ERROR_DOM_INUSE_ATTRIBUTE_ERR："Attribute already in use" 错误，因为 DOM 需要克隆属性之后才能重复使用（不像其他节点一样可以被删除）。
```

File: files/zh-cn/web/api/element/wheel_event/index.md [hunk 5]
```diff
@@ -179,13 +179,13 @@ delta 值并不代表默认情况下的实际滚动值，如果用户在滚动
...
-  - : `deltaMode` 值可以是 `DOM_DELTA_LINE` 或 `DOM_DELTA_PAGE`。然而 `Windows` 的滚轮速度设置界面和鼠标驱动工具都没有提供改为 page scroll 的选项。 所以这个值通常为 `DOM_DELTA_LINE`.
+  - : `deltaMode` 值可以是 `DOM_DELTA_LINE` 或 `DOM_DELTA_PAGE`。然而 `Windows` 的滚轮速度设置界面和鼠标驱动工具都没有提供改为 page scroll 的选项。所以这个值通常为 `DOM_DELTA_LINE`.
 - WM_GESTURE (Only when caused by panning)
-  - : `deltaMode` 值总是 `DOM_DELTA_PIXEL。`但请注意大多数笔记本的触摸板都在模拟鼠标滚轮事件而不是调用这个事件， WM_GESTURE 事件通常被平板电脑使用。
+  - : `deltaMode` 值总是 `DOM_DELTA_PIXEL。`但请注意大多数笔记本的触摸板都在模拟鼠标滚轮事件而不是调用这个事件，WM_GESTURE 事件通常被平板电脑使用。
 
 在 Mac 下 deltaMode 值由设备决定。如果设备支持高分辨率像素级滚动，`deltaMode`值就是典型的 `DOM_DELTA_PIXEL`. 然而用户可以通过加前缀`"mousewheel.enable_pixel_scrolling"`将其改变为 `DOM_DELTA_LINE` 。如果设备不支持高分辨率滚动，那么 deltaModel 值将一直为 `DOM_DELTA_LINE`.
 
-`在其它平台下， deltaMode 值总是 DOM_DELTA_LINE`.
+`在其它平台下，deltaMode 值总是 DOM_DELTA_LINE`.
```
Comment (yin1999) [inline files/zh-cn/web/api/element/wheel_event/index.md:188]:
```suggestion
在其它平台下，`deltaMode` 值总是 `DOM_DELTA_LINE`。
```

File: files/zh-cn/web/api/element/wheel_event/index.md [hunk 6]
```diff
@@ -179,13 +179,13 @@ delta 值并不代表默认情况下的实际滚动值，如果用户在滚动
 - WM_MOUSEWHEEL (竖直方向的滚动事件)
   - : `deltaMode` 值可以是 `DOM_DELTA_LINE` 或 `DOM_DELTA_PAGE。`它取决于 Windows 的用户设置 (默认设置为 `DOM_DELTA_LINE`)。
 - WM_MOUSEHWHEEL (水平方向的滚动事件)
-  - : `deltaMode` 值可以是 `DOM_DELTA_LINE` 或 `DOM_DELTA_PAGE`。然而 `Windows` 的滚轮速度设置界面和鼠标驱动工具都没有提供改为 page scroll 的选项。 所以这个值通常为 `DOM_DELTA_LINE`.
+  - : `deltaMode` 值可以是 `DOM_DELTA_LINE` 或 `DOM_DELTA_PAGE`。然而 `Windows` 的滚轮速度设置界面和鼠标驱动工具都没有提供改为 page scroll 的选项。所以这个值通常为 `DOM_DELTA_LINE`.
 - WM_GESTURE (Only when caused by panning)
-  - : `deltaMode` 值总是 `DOM_DELTA_PIXEL。`但请注意大多数笔记本的触摸板都在模拟鼠标滚轮事件而不是调用这个事件， WM_GESTURE 事件通常被平板电脑使用。
+  - : `deltaMode` 值总是 `DOM_DELTA_PIXEL。`但请注意大多数笔记本的触摸板都在模拟鼠标滚轮事件而不是调用这个事件，WM_GESTURE 事件通常被平板电脑使用。
```
Comment (yin1999) [inline files/zh-cn/web/api/element/wheel_event/index.md:184]:
```suggestion
  - : `deltaMode` 值总是 `DOM_DELTA_PIXEL`。但请注意大多数笔记本的触摸板都在模拟鼠标滚轮事件而不是调用这个事件，WM_GESTURE 事件通常被平板电脑使用。
```

File: files/zh-cn/web/api/file_and_directory_entries_api/introduction/index.md [hunk 7]
```diff
@@ -152,7 +152,7 @@ slug: Web/API/File_and_Directory_Entries_API/Introduction
 
 你不能在本地通过 file://来运行你的应用。如果你那么做了，浏览器将抛出错误，或者你的应用会静默地失败。这一限制也同样针对许多其他的文件 API，包括 BlobBuilder 和 FileReader。
 
-出于测试的目的，你可以在 Chrome 中通过在启动时添加`--allow-file-access-from-files参数来绕开这一限制，这一参数仅用于这个目的。`
+出于测试的目的，你可以在 Chrome 中通过在启动时添加`--allow-file-access-from-files 参数来绕开这一限制，这一参数仅用于这个目的。`
```
Comment (yin1999) [inline files/zh-cn/web/api/file_and_directory_entries_api/introduction/index.md:155]:
```suggestion
出于测试的目的，你可以在 Chrome 中通过在启动时添加 `--allow-file-access-from-files` 参数来绕开这一限制，这一参数仅用于这个目的。
```

File: files/zh-cn/web/api/formdata/append/index.md [hunk 8]
```diff
@@ -23,13 +23,13 @@ formData.append(name, value, filename);
 ### 参数
 
 - `name`
-  - : `value中包含的数据对应的表单名称。`
+  - : `value 中包含的数据对应的表单名称。`
```
Comment (yin1999) [inline files/zh-cn/web/api/formdata/append/index.md:26]:
```suggestion
  - : `value` 中包含的数据对应的表单名称。
```

File: files/zh-cn/web/api/formdata/formdata/index.md [hunk 9]
```diff
@@ -34,7 +34,7 @@ var formData = new FormData(); // 当前为空
 formData.append('username', 'Chris');
 ```
 
-或者你可以使用可选的`form参数来创建一个带预置数据的FormData对象`:
+或者你可以使用可选的`form 参数来创建一个带预置数据的 FormData 对象`:
```
Comment (yin1999) [inline files/zh-cn/web/api/formdata/formdata/index.md:37]:
```suggestion
或者你可以使用可选的 `form` 参数来创建一个带预置数据的 FormData 对象：
```

File: files/zh-cn/web/api/formdata/get/index.md [hunk 10]
```diff
@@ -5,7 +5,7 @@ slug: Web/API/FormData/get
 
 {{APIRef("XMLHttpRequest")}}
 
-{{domxref("FormData")}}的`get()方法用于返回FormData对象中和指定的键关联的第一个值，如果你想要返回和指定键关联的全部值，那么可以使用`{{domxref("FormData.getAll()","getAll()")}}方法。
+{{domxref("FormData")}}的`get() 方法用于返回 FormData 对象中和指定的键关联的第一个值，如果你想要返回和指定键关联的全部值，那么可以使用`{{domxref("FormData.getAll()","getAll()")}}方法。
```
Comment (yin1999) [inline files/zh-cn/web/api/formdata/get/index.md:8]:
```suggestion
{{domxref("FormData")}} 的 `get()` 方法用于返回 FormData 对象中和指定的键关联的第一个值，如果你想要返回和指定键关联的全部值，那么可以使用 {{domxref("FormData.getAll()","getAll()")}} 方法。
```

File: files/zh-cn/web/api/formdata/get/index.md [hunk 11]
```diff
@@ -39,7 +39,7 @@ formData.append('username', 'Chris');
 formData.append('username', 'Bob');
 ```
 
-`接下来使用get()来返回第一个和"username"关联的值`:
+`接下来使用 get() 来返回第一个和"username"关联的值`:
```
Comment (yin1999) [inline files/zh-cn/web/api/formdata/get/index.md:42]:
```suggestion
接下来使用 `get()` 来返回第一个和 `"username"` 关联的值：
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**